packages/gnome-shell/src/extensions/sharedInternals.d.ts

```diff
@@ -151,14 +151,10 @@ export class GettextWrapper {
 }
 
 /**
- * @see https://gitlab.gnome.org/GNOME/gnome-shell/-/blob/main/js/extensions/sharedInternals.js#L9
+ * @see https://gitlab.gnome.org/GNOME/gnome-shell/-/blob/main/js/extensions/sharedInternals.js#L285
  * @version 48
  */
-declare class Console {
-    #extension: ExtensionBase;
-
-    constructor(ext: ExtensionBase);
-
+export interface ConsoleLike {
```

Comment:
That's not enough, since this internals module isn't exported.

You need to re-export this type from `extension. d.ts` to make it available.